Parsedown.php

```diff
@@ -712,9 +712,9 @@ protected function blockRule($Line)
     #
     # Setext
 
-    protected function blockSetextHeader($Line, array $Block = null)
+    protected function blockSetextHeader($Line, array $Block = array())
```

Comment:
This is a BC break as the method can now longer be called with `null` as the second argument anymore.